### DISCUSSION TITLE
How can the SARI metric be used in a Seq2SeqTrainer compute_metric parameter

### DISCUSSION THREAD

**Discussion by eytan-c** [huggingface/datasets#2712] [Q&A]
I want to use the sair metic (https://github.com/huggingface/datasets/pull/1875#issue-573036714) as the metric for the trainer, but am stuck on how it can be used, since sari expectes `sources`, `predictions`, and `references` as inputs, while an `EvalPrediction` object only has the latter two.

Thanks!

**Comment by mariosasko** [comment]
Hi! This limitation has been addressed recently in https://github.com/huggingface/transformers/pull/16461, so now you can pass the `sources` values as `inputs` to the `EvalPrediction` object.
